### PR TITLE
[MM-60857] Replace FormattedMarkdownMessage in 'webapp/channels/src/components/plugin_marketplace/marketplace_item/marketplace_item_plugin/marketplace_item_plugin.tsx' with FormattedMessage

### PR DESCRIPTION
**Pull request by Aryakoste**
#### Ticket Link

Fixes https://github.com/mattermost/mattermost/issues/28697
Jira https://mattermost.atlassian.net/browse/MM-60857
#### Release Note

```release-note
NONE
```

### CODE REVIEW

**Review by M-ZubairAhmed — CHANGES_REQUESTED** [review]
Can you check failing ci

File: webapp/channels/src/components/plugin_marketplace/marketplace_item/marketplace_item_plugin/marketplace_item_plugin.tsx [hunk 1]
```diff
@@ -131,10 +130,19 @@ export const UpdateConfirmationModal = ({show, name, version, installedVersion,
...
                     id='marketplace_modal.list.update_confirmation.message.current_with_release_notes'
-                    defaultMessage='You currently have {installedVersion} installed. View the [release notes](!{releaseNotesUrl}) to learn about the changes included in this update.'
-                    values={{installedVersion, releaseNotesUrl}}
+                    defaultMessage='You currently have {installedVersion} installed. View the <a>release notes</a> to learn about the changes included in this update.'
+                    values={{
+                        installedVersion,
+                        a: (chunks: string) => (
+                            <ExternalLink
+                                href={releaseNotesUrl as string}
+                                location='release-notes-modal'
```
Comment (M-ZubairAhmed) [inline webapp/channels/src/components/plugin_marketplace/marketplace_item/marketplace_item_plugin/marketplace_item_plugin.tsx:141]:
```suggestion
                                location='plugin-marketplace'
```

File: webapp/channels/src/components/plugin_marketplace/marketplace_item/marketplace_item_plugin/marketplace_item_plugin.tsx [hunk 2]
```diff
@@ -165,10 +173,18 @@ export const UpdateConfirmationModal = ({show, name, version, installedVersion,
...
+                    <FormattedMessage
                         id='marketplace_modal.list.update_confirmation.message.warning_major_version_with_release_notes'
-                        defaultMessage='This update may contain breaking changes. Consult the [release notes](!{releaseNotesUrl}) before upgrading.'
-                        values={{releaseNotesUrl}}
+                        defaultMessage='This update may contain breaking changes. Consult the <a>release notes</a> before upgrading.'
+                        values={{
+                            a: (chunks: string) => (
+                                <ExternalLink
+                                    href={releaseNotesUrl as string}
+                                    location='release-notes-modal'
```
Comment (M-ZubairAhmed) [inline webapp/channels/src/components/plugin_marketplace/marketplace_item/marketplace_item_plugin/marketplace_item_plugin.tsx:183]:
```suggestion
                                    location='plugin-marketplace'
```

**Review by M-ZubairAhmed — CHANGES_REQUESTED** [review]
Lets change these values to align with correct utm

**Review by M-ZubairAhmed — APPROVED** [review]
This looks good :shipit: Thank you

**Review by larkox — APPROVED** [review]

**Comment by Rutam21** [comment]
@lindy65 @M-ZubairAhmed Please assign this PR to me for QA Testing. Thanks.

**Comment by lindy65** [comment]
Assigned to you for testing @Rutam21 👍

**Comment by Rutam21** [comment]
### This PR has been tested in the Test Server, and all the intended changes are working as expected.

- There are no visual changes in the UI and the rich text formatting in the affected areas is preserved.
- There has been no change in the functionality of this section since the component update.

Please find the comparison snapshots of this Test Server with another without these changes.

#### Another Test Server (Without these changes)

ThisPRTestServerMajorUpdate

#### Current PR Test Server

CurrentPRTestServerMajorUpdate

The above snapshots confirm the Major Update confirmation dialog.

cc: @lindy65

**Comment by lindy65** [comment]
Thanks for testing @Rutam21 - counted towards Hacktoberfest

cc @M-ZubairAhmed

**Pull request merged**